### PR TITLE
Added KeyFM

### PR DESCRIPTION
**Pull request by jackkelly-bayliss**
KeyFM-PreMiD

### CODE REVIEW

File: websites/K/KeyFM/presence.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  }
+
+  if (changesmalltext) {
+    presenceData.smallImageText = changesmalltext.replace('%listeners%', keylisteners);
+  } else {
+    presenceData.smallImageText = "Listeners: " + keylisteners;
+  }
+
+    presence.setActivity(presenceData, true);
+    presence.setTrayTitle();
```
Comment (callumok2004) [inline websites/K/KeyFM/presence.ts:93]: Not required - Unless used when not sending any data (If I am correct)

Comment (callumok2004) [reply websites/K/KeyFM/presence.ts:93]: presence.setTrayTitle()

Comment (jackkelly-bayliss) [reply websites/K/KeyFM/presence.ts]: Committed, thank you Callum :P

Comment (callumok2004) [reply websites/K/KeyFM/presence.ts]: presence.setTrayTitle(); //Clears the tray title for mac users

File: websites/K/KeyFM/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    "id": "140590672657776641"
+  },
+  "service": "KeyFM",
+  "description": {
+    "en": "KeyFM.net is a non-profit radio community providing music around the globe, all day - every day."
+  },
+  "url": [
+    "keyfm.net",
+    "keyfm.net/splash",
+    "keyfm.net/splash/Key.Home"
```
Comment (Bas950) [inline websites/K/KeyFM/dist/metadata.json:13]: remove these are incorrect

Comment (frozaum) [inline websites/K/KeyFM/dist/metadata.json:13]: pls remove them

Comment (jackkelly-bayliss) [reply websites/K/KeyFM/dist/metadata.json]: Removed thank you.

File: websites/K/KeyFM/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "name": "Jack Hansen",
+    "id": "140590672657776641"
+  },
+  "service": "KeyFM",
+  "description": {
+    "en": "KeyFM.net is a non-profit radio community providing music around the globe, all day - every day."
+  },
+  "url": [
+    "keyfm.net"
+  ],
```
Comment (Bas950) [inline websites/K/KeyFM/dist/metadata.json:12]: its only 1 url now so it doesnt need to be in an array

Comment (jackkelly-bayliss) [reply websites/K/KeyFM/dist/metadata.json:12]: Good spot! Apologies for that one, I had to edit before heading to work. Shall edit that one now.

**Pull request merged**